da/avail/datasubmit/submitdata.go

```diff
@@ -0,0 +1,88 @@
+package datasubmit
+
+import (
+	"errors"
+
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client/v4"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/signature"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+)
+
+// submit data submits the extrinsic through substrate api
+func SubmitData(apiURL string, seed string, appID int, data []byte) error {
+
+	// if app id is greater than 0 then it must be created before submitting data
+	if appID == 0 {
```

Comment:
Does the following makes sense?
```suggestion
	if appID < 1 {
```